web/Web/web/static/js/document-viewer.js

```diff
@@ -1259,14 +1257,14 @@ docView.prototype = {
     return this.triggerEvent("beforeDocumentLoad", [docid]);
   },
 
-  afterDocumentLoad: function(docid, rel) {
+  afterDocumentLoad: function(docid) {
     "use strict";
-    return this.triggerEvent("afterDocumentLoad", [docid, rel]);
```

Comment:
You can return the rel value here, but extract it in the function itself rather than passing it as a parameter. That's what I meant earlier.

Reply:
Done!